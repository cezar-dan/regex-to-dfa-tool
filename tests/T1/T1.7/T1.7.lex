 
SPACE
0
0,' ',1
1,' ',2
1

abcde
TOKEN1
0
0,'a',1
0,'b',2
0,'c',3
0,'d',3
0,'e',3
1,'a',1
1,'b',2
1,'c',4
1,'d',5
1,'e',3
2,'a',1
2,'b',2
2,'c',4
2,'d',5
2,'e',3
4,'a',3
4,'b',3
4,'c',3
4,'d',3
4,'e',6
5,'a',3
5,'b',3
5,'c',3
5,'d',3
5,'e',6
6,'a',3
6,'b',3
6,'c',3
6,'d',3
6,'e',3
6

abcde
TOKEN2
0
0,'a',1
0,'b',2
0,'c',3
0,'d',2
0,'e',4
1,'a',2
1,'b',5
1,'c',2
1,'d',2
1,'e',2
3,'a',2
3,'b',2
3,'c',2
3,'d',6
3,'e',2
4,'a',2
4,'b',2
4,'c',2
4,'d',2
4,'e',2
5,'a',1
5,'b',2
5,'c',3
5,'d',2
5,'e',4
6,'a',2
6,'b',2
6,'c',2
6,'d',6
6,'e',2
4 6 3

abde
TOKEN3
0
0,'a',1
0,'b',2
0,'d',1
0,'e',1
2,'a',3
2,'b',2
2,'d',4
2,'e',5
3,'a',3
3,'b',1
3,'d',1
3,'e',5
4,'a',3
4,'b',1
4,'d',4
4,'e',5
5,'a',3
5,'b',1
5,'d',1
5,'e',5
4 5 3 2

bcde
TOKEN4
0
0,'b',1
0,'c',2
0,'d',2
0,'e',3
1,'b',2
1,'c',5
1,'d',2
1,'e',2
3,'b',2
3,'c',2
3,'d',4
3,'e',2
4,'b',1
4,'c',2
4,'d',2
4,'e',3
5,'b',1
5,'c',2
5,'d',2
5,'e',3
4 5

abcde
TOKEN5
0
0,'a',1
0,'b',2
0,'c',3
0,'d',4
0,'e',5
1,'a',5
1,'b',5
1,'c',5
1,'d',5
1,'e',7
2,'a',1
2,'b',2
2,'c',3
2,'d',4
2,'e',5
3,'a',1
3,'b',2
3,'c',3
3,'d',4
3,'e',5
4,'a',6
4,'b',5
4,'c',5
4,'d',5
4,'e',5
6,'a',1
6,'b',5
6,'c',5
6,'d',4
6,'e',5
7,'a',1
7,'b',5
7,'c',5
7,'d',4
7,'e',5
6 7