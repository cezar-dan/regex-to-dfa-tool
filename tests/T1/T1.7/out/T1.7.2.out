SPACE  
TOKEN2 c
TOKEN1 aaace
SPACE  
TOKEN4 edbcededbcedbcbc